tests/containerfiles/dummyscenario/krknctl-input.json

```diff
@@ -31,5 +31,16 @@
     "type":"string",
     "required": "false",
     "secret": "true"
-  }
+  },
+  {
+  "name": "krkn-run-mode",
```

Comment:
unfortunately that's the wrong place, this is the dummy scenario containerfile, you have to open a PR in the krkn source repo from where the image is built: 

https://github.com/krkn-chaos/krkn/blob/main/containers/krknctl-input.json